diff --git a/src/linyaps_box/container.cpp b/src/linyaps_box/container.cpp
index 9cca48d..932fc9a 100644
--- a/src/linyaps_box/container.cpp
+++ b/src/linyaps_box/container.cpp
@@ -1251,6 +1251,14 @@ void configure_mounts(const linyaps_box::container &container, const std::filesy
         return ss.str();
     }();
 
+    // NOTE: This node only exist in UOS kernel, it is used to identify child process
+    // running in linyaps container
+    auto elf_verify_cred = std::filesystem::path{ "/proc/self/attr/elfverify/cred" };
+    if (std::filesystem::exists(elf_verify_cred)) {
+        std::ofstream ofs(elf_verify_cred);
+        ofs << "1";
+    }
+
     execvpe(c_args[0],
             const_cast<char *const *>(c_args.data()), // NOLINT
             const_cast<char *const *>(c_env.data())); // NOLINT
